src/transform/WidenImage.js

```diff
@@ -1,30 +1,99 @@
 import './WidenImage.css'
```

Comment:
This change involves a significant refactor and functionality change. If you don't mind, please update our tests for the new functionality and add a link to one of the articles that exhibits the problem.

Reply:
Tests added!